### ISSUE TITLE
[FromQuery] with custom class problem

### ISSUE THREAD

**Issue by SuperJMN** [paulcbetts/refit#370]
Hello, friends of Refit! First of all, thanks for such a cool project. It's REALLY useful.

I'm having a problem with a Get method in my Web API: The API gets the object but with the default values. 

For instance: 

`service.GetSummary(new MyClass() { Prop = 1 });`

The Web API receives a MyClass intance, but Prop is 0!

This is all that I have.

The Get method (Controller in the Web API):

```
        [HttpGet]
        public async Task<ActionResult> Get([FromQuery]MyClass req)
```

MyClass is:

```
    public class MyClass
    {
        public int Prop { get; set; }
    }
```

and my Web API interface for Refit is:

```
    public interface IMyWebApi
    {
        [Get("/api/interinos")]
        Task<PositionSummary> GetSummary(MyClass req);
    }
```

upon the call

service.GetSummary(new MyClass() { Prop = 1 });

I'm getting a MyClassInstance in my controller, but Prop is 0, instead of 1.

What am I doing wrong?

**Comment by Cheesebaron** [comment]
Why are you using `[FromQuery]` when you send the contents as the body of the request? Doesn't the FromQueryAttribute indicate that the contents come in the url, in this case as the `req` query string parameter?

**Comment by bennor** [comment]
I don't think `FromQuery` is a Refit attribute.

**Comment by bennor** [comment]
So this is a duplicate of #315 and is addressed by #355.
